cmd/network/verification_test.go

```diff
@@ -142,6 +147,18 @@ func Test_egressVerificationGenerateAWSValidateEgressInput(t *testing.T) {
 							},
 						},
 					},
+					describeRouteTablesResp: &ec2.DescribeRouteTablesOutput{
```

Comment:
I think a few other tests also need this `describeRouteTablesResp` property defined now so they can pass.

Reply:
Done✅